Missing dependency "@angular/cdk"

When running a clean instance of `mdm-explorer` and after running `npm install`, there is still a compiler error about a missing dependency:

```
Error: src/app/app.component.ts:33:34 - error TS2307: Cannot find module '@angular/cdk/overlay' or its corresponding type declarations.

33 import { OverlayContainer } from '@angular/cdk/overlay';
```

The `@angular/cdk` package is missing from the dependency list, which `@angular/material` depends on.

To fix this, run `npm install --save-exact @angular/cdk@13.3.8`, ensuring that the dependency is pinned.

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
   "dependencies": {
     "@angular-material-extensions/password-strength": "10.1.1",
     "@angular/animations": "13.3.10",
+    "@angular/cdk": "13.3.8",
     "@angular/common": "13.3.10",
     "@angular/compiler": "13.3.10",
     "@angular/core": "13.3.10",

